src/clients/clone_system.rb

```diff
@@ -23,11 +23,21 @@ def main
       Yast.import "Y2ModuleConfig"
       Yast.import "Mode"
       Yast.import "FileUtils"
+      Yast.import "Stage"
+      Yast.import "Report"
 
       textdomain "autoinst"
 
       @moduleList = ""
 
+      if Mode.normal && Stage.normal && !PackageSystem.Installed("autoyast2")
+        ret = PackageSystem.InstallAll(["autoyast2"])
+        # The modules/clients has to be reloaded. That's why the user has to
+        # restart the export again.
+        Report.Message(_("Please restart the clone process.")) if ret
```

Comment:
You can automatically restart Yast, just create `/var/lib/YaST2/restart_yast` file and finish the client. The main script will start the client again, see https://github.com/yast/yast-yast2/blob/master/scripts/yast2#L414.

But it seems you have to explicitly remove that file after restart to avoid endless loop... Please, check that.